scripts/globals/spells/trust/excenmille.lua

```diff
@@ -41,9 +41,7 @@ function onMobSpawn(mob)
     mob:addSimpleGambit(ai.t.SELF, ai.c.TP_GTE, 1000,
                         ai.r.WS, ai.s.SPECIFIC, tpz.ws.DOUBLE_THRUST)
 
-    -- Excenmille is a PLD who uses a Polearm, so raise his MAIN_DMG_RATING (up from 1H Sword levels)
-    local increase_damage_by_percent = 30
-    mob:addMod(tpz.mod.MAIN_DMG_RATING, mob:getWeaponDmg() * (1.0 + (increase_damage_by_percent / 100)))
+    mob:addMod(tpz.mod.STORETP, 25)
```

Comment:
If his tp per hit isn't lining up, are we sure the base delay is correct or that he doesn't secretly sub sam?

Reply:
I'll have to sit with him in retail to make sure his TP stats line up

Reply:
(This is based off a SE update where they said they added the "Store TP Trait" to Excenmille, rather than trying to balance him properly)

Reply:
Ahh, I was unaware we had a word of god source for his storeTP. :+1: